docs/Developer-guide/Development-Workflow.md

```diff
@@ -294,8 +306,9 @@ changes in code, you can mark each of the inline comment as 'done'
 (optional). After all the changes to your local files, amend the
 previous commit with these changes with -
 
-        sh$ git commit -a --amend
-
+```console
+# git commit -a --amend
```

Comment:
I feel adding all the files directly during a commit is a bad practice to encourage. Shall we ask them to add and amend?

Reply:
Can be separate PR. I want to avoid too deep review for this PR, content changes can be addressed in different PR?

Reply:
makes sense.